util/buildbot/buildwin32.sh

```diff
@@ -33,22 +33,21 @@ irrlicht_version=$(cat $topdir/../../misc/irrlichtmt_tag.txt)
 
 mkdir -p $libdir
 
-# 'dw2' just points to rebuilt versions after a toolchain change
-# this distinction should be gotten rid of next time
+# 'ucrt' just points to rebuilt versions after a toolchain change
 
 cd $libdir
 libhost="http://minetest.kitsunemimi.pw"
-download "https://github.com/minetest/irrlicht/releases/download/$irrlicht_version/win32.zip" irrlicht-$irrlicht_version-win32.zip
+download "https://github.com/minetest/irrlicht/releases/download/$irrlicht_version/win32-ucrt.zip" irrlicht-$irrlicht_version-win32.zip
```

Comment:
What is the origin of this file name? If it is packed and  named automatically, https://github.com/minetest/irrlicht/pull/260 might need adjustments.

Reply:
Release artifacts are uploaded (and renamed) by hand so shouldn't be an issue.

Reply:
Packing it with the right name for uploading might be less error-prone but this is fine by me because it's not updated often anyway :shrug: . Thanks for clarifying.